app/models/invitation.rb

```diff
@@ -0,0 +1,10 @@
+class Invitation < ApplicationRecord
+  belongs_to :workspace
+  belongs_to :receiver, class_name: 'User', optional: true
+
+  validates :token, presence: true, uniqueness: true
+
+  def active?
+    active
+  end
```

Comment:
if `active` is a boolean column, ActiveRecord will already provide this `active?` method for you, so you don't need to set it manually.

Reply:
dah! Done!